src/types/obiTypes.ts

```diff
@@ -122,6 +122,11 @@ export interface OBIDialog {
      * The property to set the value of
      */
     property?: string;
+    /**
+     * If the result is expected to be structured data such as a JSON object, this holds
+     * the names of the fields or properties in that response object.
+     */
+    responseFields?: string[]
```

Comment:
nit - Might not here that this is an added field as the comment at top of file says that it's generated.  If it is re-generated at some point, we'll lose this field

Reply:
Done.